server/src/main/resources/application-local.yml

```diff
@@ -1,66 +1,5 @@
-micronaut:
-  application:
-    name: check-ins
-
-  server:
-    cors:
-      enabled: true
-    maxRequestSize: 100MB
-    multipart:
-      maxFileSize: 100MB
-
-  router:
-    static-resources:
```

Comment:
why did we delete all the config from application local ? is it for testing ?

Reply:
These files are cascading. You don't need to redefine all of the properties...only the ones that need to change. Check out the micronaut configuration documentation, it talks about order of precedence for properties.

Reply:
No, I meant swagger properties from local yaml. I don't see them defined anywhere.

Reply:
sorry, should have viewed the complete application.yaml and not just the changes :P